### ISSUE TITLE
craft\services\Elements::getElementById(): Argument #1 ($elementId) must be of type int, null given, called in /var/task/vendor/twig/twig/src/Extension/CoreExtension.php on line 1802

### ISSUE THREAD

**Issue by jamesmacwhite** [verbb/formie#2150]
### Describe the bug

I am getting the following error when accessing a form from the Control Panel since 3.0.11

> craft\services\Elements::getElementById(): Argument #1 ($elementId) must be of type int, null given, called in /var/task/vendor/twig/twig/src/Extension/CoreExtension.php on line 1802

Full stacktrace

```
TypeError: craft\services\Elements::getElementById(): Argument #1 ($elementId) must be of type int, null given, called in /var/task/vendor/twig/twig/src/Extension/CoreExtension.php on line 1802 and defined in /var/task/vendor/craftcms/cms/src/services/Elements.php:899
Stack trace:
#0 /var/task/vendor/twig/twig/src/Extension/CoreExtension.php(1802): craft\services\Elements->getElementById(NULL, 'craft\\elements\\...', 1)
#1 /var/task/vendor/craftcms/cms/src/helpers/Template.php(148): Twig\Extension\CoreExtension::getAttribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(craft\services\Elements), 'getElementById', Array, 'method', false, false, false, 13)
#2 /tmp/craft-storage/runtime/compiled_templates/b2/b2d90a37853dac213bdcc65efa7b7251.php(189): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(craft\services\Elements), 'getElementById', Array, 'method', false, false, false, 13)
#3 /var/task/vendor/twig/twig/src/Extension/CoreExtension.php(1967): __TwigTemplate_31e7741217125d6db5f56db6768644a8->{closure}()
#4 /tmp/craft-storage/runtime/compiled_templates/b2/b2d90a37853dac213bdcc65efa7b7251.php(155): Twig\Extension\CoreExtension::captureOutput(Object(Generator))
#5 /var/task/vendor/twig/twig/src/Extension/CoreExtension.php(1309): __TwigTemplate_31e7741217125d6db5f56db6768644a8->macro_tableColumns(Object(craft\elements\Entry), Object(craft\fields\Matrix), Array)
#6 /tmp/craft-storage/runtime/compiled_templates/b2/b2d90a37853dac213bdcc65efa7b7251.php(200): Twig\Extension\CoreExtension::callMacro(Object(__TwigTemplate_31e7741217125d6db5f56db6768644a8), 'macro_tableColu...', Array, 17, Array, Object(Twig\Source))
#7 /var/task/vendor/twig/twig/src/Extension/CoreExtension.php(1967): __TwigTemplate_31e7741217125d6db5f56db6768644a8->{closure}()
#8 /tmp/craft-storage/runtime/compiled_templates/b2/b2d90a37853dac213bdcc65efa7b7251.php(155): Twig\Extension\CoreExtension::captureOutput(Object(Generator))
#9 /var/task/vendor/twig/twig/src/Extension/CoreExtension.php(1309): __TwigTemplate_31e7741217125d6db5f56db6768644a8->macro_tableColumns(Object(craft\elements\Entry), Object(craft\fields\Matrix), Array)
#10 /tmp/craft-storage/runtime/compiled_templates/b2/b2d90a37853dac213bdcc65efa7b7251.php(200): Twig\Extension\CoreExtension::callMacro(Object(__TwigTemplate_31e7741217125d6db5f56db6768644a8), 'macro_tableColu...', Array, 17, Array, Object(Twig\Source))
#11 /var/task/vendor/twig/twig/src/Extension/CoreExtension.php(1967): __TwigTemplate_31e7741217125d6db5f56db6768644a8->{closure}()
#12 /tmp/craft-storage/runtime/compiled_templates/b2/b2d90a37853dac213bdcc65efa7b7251.php(155): Twig\Extension\CoreExtension::captureOutput(Object(Generator))
#13 /var/task/vendor/twig/twig/src/Extension/CoreExtension.php(1309): __TwigTemplate_31e7741217125d6db5f56db6768644a8->macro_tableColumns(Object(craft\elements\Entry), Object(verbb\formie\fields\Forms), Array)
#14 /tmp/craft-storage/runtime/compiled_templates/b2/b2d90a37853dac213bdcc65efa7b7251.php(99): Twig\Extension\CoreExtension::callMacro(Object(__TwigTemplate_31e7741217125d6db5f56db6768644a8), 'macro_tableColu...', Array, 79, Array, Object(Twig\Source))
#15 /var/task/vendor/twig/twig/src/Template.php(393): __TwigTemplate_31e7741217125d6db5f56db6768644a8->doDisplay(Array, Array)
#16 /tmp/craft-storage/runtime/compiled_templates/60/6089932d4abaf3831b1b5deda73d038b.php(284): Twig\Template->yield(Array)
#17 /var/task/vendor/twig/twig/src/Template.php(437): __TwigTemplate_b0aba3c94d787597e9793c0f33dbe8e4->block_content(Array, Array)
#18 /tmp/craft-storage/runtime/compiled_templates/2d/2de72dedf09eb9113c0802805a2144f0.php(672): Twig\Template->yieldBlock('content', Array, Array)
#19 /var/task/vendor/twig/twig/src/Template.php(437): __TwigTemplate_3ee0957a018eb1aecbc27559842caf27->block_main(Array, Array)
#20 /tmp/craft-storage/runtime/compiled_templates/2d/2de72dedf09eb9113c0802805a2144f0.php(399): Twig\Template->yieldBlock('main', Array, Array)
#21 /var/task/vendor/twig/twig/src/Template.php(437): __TwigTemplate_3ee0957a018eb1aecbc27559842caf27->block_body(Array, Array)
#22 /tmp/craft-storage/runtime/compiled_templates/90/907289f28172fb5817ef82424fa8f4f1.php(105): Twig\Template->yieldBlock('body', Array, Array)
#23 /var/task/vendor/twig/twig/src/Template.php(393): __TwigTemplate_52b8fdb2d416153858f82ef7e36e96cc->doDisplay(Array, Array)
#24 /tmp/craft-storage/runtime/compiled_templates/5a/5afaef5bb06bb466d5b9d21f615eabf3.php(61): Twig\Template->yield(Array, Array)
#25 /var/task/vendor/twig/twig/src/Template.php(393): __TwigTemplate_7fb4d443d6c57c0a014d43970aee2fa4->doDisplay(Array, Array)
#26 /tmp/craft-storage/runtime/compiled_templates/2d/2de72dedf09eb9113c0802805a2144f0.php(175): Twig\Template->yield(Array, Array)
#27 /var/task/vendor/twig/twig/src/Template.php(393): __TwigTemplate_3ee0957a018eb1aecbc27559842caf27->doDisplay(Array, Array)
#28 /tmp/craft-storage/runtime/compiled_templates/60/6089932d4abaf3831b1b5deda73d038b.php(80): Twig\Template->yield(Array, Array)
#29 /var/task/vendor/twig/twig/src/Template.php(393): __TwigTemplate_b0aba3c94d787597e9793c0f33dbe8e4->doDisplay(Array, Array)
#30 /var/task/vendor/twig/twig/src/Template.php(349): Twig\Template->yield(Array, Array)
#31 /var/task/vendor/twig/twig/src/Template.php(364): Twig\Template->display(Array)
#32 /var/task/vendor/twig/twig/src/TemplateWrapper.php(35): Twig\Template->render(Array)
#33 /var/task/vendor/twig/twig/src/Environment.php(306): Twig\TemplateWrapper->render(Array)
#34 /var/task/vendor/craftcms/cms/src/web/View.php(539): Twig\Environment->render('formie/forms/_e...', Array)
#35 /var/task/vendor/craftcms/cms/src/web/View.php(592): craft\web\View->renderTemplate('formie/forms/_e...', Array)
#36 /var/task/vendor/craftcms/cms/src/web/TemplateResponseFormatter.php(57): craft\web\View->renderPageTemplate('formie/forms/_e...', Array, 'cp')
#37 /var/task/vendor/yiisoft/yii2/web/Response.php(1109): craft\web\TemplateResponseFormatter->format(Object(craft\web\Response))
#38 /var/task/vendor/craftcms/cms/src/web/Response.php(339): yii\web\Response->prepare()
#39 /var/task/vendor/yiisoft/yii2/web/Response.php(340): craft\web\Response->prepare()
#40 /var/task/vendor/yiisoft/yii2/base/Application.php(390): yii\web\Response->send()
#41 /var/task/public/index.php(12): yii\base\Application->run()
#42 {main}
```

It appears to be related to some forms but not all, so I'm not sure what the specific issue is.

I noticed a user permission commit has been recently made, I've also tried this with a Craft admin user and it's the same.

The front end is displaying the form linked through the form field without errors, although the address field is missing on the front end.

### Steps to reproduce

1. Go to `/formie/forms`
2. Select a form with fields
3. The above error is thrown

### Form settings

- Multi-page form: Yes or No
- Submission Method: Ajax or Page Reload
- Client-side Validation: Yes or No
- Custom Form Templates: Yes or No

### Craft CMS version

5.5.0.1

### Plugin version

3.0.11

### Multi-site?

N/A

### Additional context

N/A

**Comment by jamesmacwhite** [comment]
I suspect this commit, may be involved: https://github.com/verbb/formie/commit/f717b7b16e67a2c46f735dfdc2810b668f22ad17

Downgrading back to 3.0.10 resolves the issue.

**Comment by engram-design** [comment]
Sorry about that, fixed one problem - cause another! Fixed in [3.0.12](https://github.com/verbb/formie/releases/tag/3.0.12)

**Comment by jamesmacwhite** [comment]
It happens, thanks Josh for the quick fix and release.
